packages/tooling/fast-tooling-react/src/form/custom-controls/css/color-picker-component/color-picker.styles.ts

```diff
@@ -0,0 +1,138 @@
+import { css } from "@microsoft/fast-element";
+
+export const ColorPickerStyles = css`
+
+    .root .colorUI
```

Comment:
nit - would prefer kebab case for consistency across the project. Turtles all the way down, but not a *required* change.

Reply:
Done